spk/rutorrent/src/service-setup.sh

```diff
@@ -154,7 +164,6 @@ service_save ()
     if [ -f "${WEB_DIR}/${PACKAGE}/.htaccess" ]; then
         mv "${WEB_DIR}/${PACKAGE}/.htaccess" "${TMP_DIR}/" >>"${INST_LOG}" 2>&1
     fi
-    cp -pr ${WEB_DIR}/${PACKAGE}/share/ ${TMP_DIR}/ >>"${INST_LOG}" 2>&1
```

Comment:
Maybe a stupid question but I believe this would mean some manually installed resources would get lost. Besides, I believe the files which were "saved" here are copied back later on. No other risks foreseen?

Reply:
That would mean that.  I noticed that an issue where the share directory ended-up being recpvered to a wrong location over to `/var/package/rutorrent/target/var` for an unknown reason.  And really to me, playing with that also means being in a non-supported environment.  I believe as is all regular users are covered by default.